src/pages/fileupload/FileUploadContainer.state.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+
+import FileUpload from 'synapsefi-dev-ui/dist/components/FileUpload/FileUpload'
+import CheckboxGroup from 'synapsefi-dev-ui/dist/components/CheckboxGroup/CheckboxGroup'
+
+const FileUploadContainer = () => {
+  return(
+    <FileUpload
+        acceptedFiles={[
+          "png",
+          "jpg",
+          "jpeg",
+          "pdf"
+        ]}
+        files={[
+          {
+            name: 'https://cdn.synapsepay.com/uploads/2019/10/02/3BmuL7.png'
+          }
+        ]}
+        fileValues={{
+          'bd58154cd50dab5064b577e470f52474f0b57aa2aa33ed18d616f551b307ee03': {
+            'subDocumentId': 'bd58154cd50dab5064b577e470f52474f0b57aa2aa33ed18d616f551b307ee03',
+            'documentValue': 'https://cdn.synapsepay.com/uploads/2019/10/02/3BmuL7.png'
+          }
+        }}
+        deleteAction={() => console.log('Delete Action')}
+        fileUploadRowText={'1. Proof of account'}
+        modalHeaderText={"Delete Document"}
+        modalChildren={() => (
+            <CheckboxGroup
```

Comment:
Please be sure the indentation is only 2 spaces, thanks

Reply:
Updated all indents